src/Page/SubmitStory/View.elm

```diff
@@ -0,0 +1,110 @@
+module Page.SubmitStory.View exposing (view)
+
+import Css exposing (Style, batch, border, displayFlex, flexGrow, flexShrink, flexWrap, height, hidden, int, overflow, pct, property, rem, width, wrap)
+import Html.Styled exposing (Html, div, h2, iframe, img, text)
+import Html.Styled.Attributes as Attr exposing (css)
+import I18n.Keys exposing (Key(..))
+import I18n.Translate exposing (Language(..), translate)
+import Message exposing (Msg)
+import Page.Data
+import Shared exposing (Model)
+import Theme.Global exposing (centerContent, primaryHeader, roundedCornerStyle)
+import Theme.Markdown exposing (markdownToHtml)
+
+
+view : Model -> Page.Data.Page -> Html Msg
+view model page =
+    div [ css [ centerContent ] ]
+        [ primaryHeader [] page.title
+        , viewSubmitFlex model page
+        ]
+
+
+viewSubmitFlex : Model -> Page.Data.Page -> Html Msg
+viewSubmitFlex model page =
+    div
+        [ css
+            [ batch
+                [ displayFlex
```

Comment:
We could move these styles to a function - but I'm OK with this.

Reply:
oh no, sorry I missed that one